run.sh

```diff
@@ -104,6 +105,7 @@ if [[ ! -f /var/spool/cron/crontabs/pi ]]; then
   mv /root/crontab /var/spool/cron/crontabs/pi
 fi
 touch /etc/crontab /etc/cron.d/* /var/spool/cron/crontabs/*
+touch /etc/cron.d/emoncms
```

Comment:
Duplication of line 107

Reply:
Line 107 does not create the emoncms file

Reply:
/etc/cron.d/emoncms not required. User 'pi' cronjobs listed in /var/spool/cron/crontabs/pi - created in line 105 of it does not exist yet.